sci-libs/bliss/bliss-0.77.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+SRC_URI="https://users.aalto.fi/~tjunttil/${PN}/downloads/${P}.zip"
+DESCRIPTION="Compute Automorphism Groups and Canonical Labelings of Graphs"
+HOMEPAGE="https://users.aalto.fi/~tjunttil/bliss/index.html"
+
+LICENSE="LGPL-3"
+SLOT="0/1"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="gmp"
+
+RDEPEND="gmp? ( dev-libs/gmp:0= )"
+DEPEND="${RDEPEND}
+	app-arch/unzip"
+PATCHES=( "${FILESDIR}/${PN}-0.77-install.patch" )
+
+# No tests are currently provided upstream
+#RESTRICT="test"
```

Comment:
Just remove these comments, no need to restrict the tests if there are no tests anyway

Reply:
That, I was supposed to remove before the push. Got distracted.